ui/page/dcrdex/dex_onboarding_page.go

```diff
@@ -314,7 +314,7 @@ func (pg *DEXOnboarding) onBoardingStep(gtx C, step onboardingStep, stepDesc str
 			inset := layout.Inset{Top: u10, Bottom: u10}
 			if !activeStep {
 				return inset.Layout(gtx, func(gtx C) D {
-					return pg.semiBoldLabelGrey3(gtx, stepDesc)
+					return semiBoldLabelGrey3(pg.Theme, gtx, stepDesc)
```

Comment:
Why we need to modify this function but semiBoldLabel() function not change, it causes inconsistency

Reply:
This is shared but the other is not.